README.md

```diff
@@ -103,7 +108,7 @@ The demos use config files and we have provided a templated version of each conf
 - {repo-root-dir}/core/invehicle_digital_twin/template
 - {repo-root-dir}/samples/common/template
 
-The following instructions are for the demo for the use of a property.
+The following instructions are for the demo for the use of a property.  This sample uses a MQTT Broker; please make sure that it is running.
```

Comment:
I think you should add a subsection within the "Running the Demo" section for each demo scenario to improve the readability.

For example:

## Running the Demo

### Property
...
### Mixed use of commands and properties.
....